roottest/root/io/references/A.C

```diff
@@ -1,17 +1,17 @@
-#include "TRef.h"
-
-#ifndef A_C
-#ifdef ClingWorkAroundMultipleInclude
-#define A_C
-#endif
-
-class A : public TObject { 
-public: 
-A(A* other = 0) { _o = other; }
+#ifndef MACRO_A_C
+#define MACRO_A_C
 
-TRef _o; 
+#include "TRef.h"
 
-ClassDef(A, 1)
+class A : public TObject {
+public:
+  A(A* other = nullptr) { _o = other; }
+  TRef _o;
+  ClassDef(A, 1)
 };
 
+#ifdef __ROOTCLING__
+#pragma link C++ class A+;
+#endif
```

Comment:
The need for this is directly linked to the change indicated at: https://github.com/root-project/root/pull/19156/files#r2167236044.

In particular due to the fact that `lotsRef.C` includes `A.C`, this line means that the dictionary for 'A' is generated for/in both `A_C.so` and `lotsRefs_C.so` ...

Reply:
Yes, you right!

I add `ROOTEXE_OPTS` to `ROOTTEST_COMPILE_MACRO` and remove such extra entry